recipe/meta.yaml

```diff
@@ -43,6 +46,8 @@ requirements:
   run:
     - python
     - qdldl-python
+    - jinja2
```

Comment:
why would osqp run-depend on jinja? 🤔

Reply:
Not 100% sure, but it is a dependency upstream: https://github.com/osqp/osqp-python/blob/97ad92ea3d16b330bb9b9abc95eeaca83439a9ad/pyproject.toml#L19

maybe it's used in the codegen part?

Reply:
ok, thx